roles/ceph-common/defaults/main.yml

```diff
@@ -68,9 +68,8 @@ ceph_common:
 
   ## OSD options
   #
-  journal_size: 10000 # 10 GB
-  pool_default_pg_num: 256
-  pool_default_pgp_num: 256
+  journal_size: 5000 # 5 GB
+  pool_name: default

```

Comment:
i think this is the source of the merge conflict. it is because i merged from bluebox/master at some point, when i probably shouldn't have. i think you want to take the upstream changes.